streaming/streaming.go

```diff
@@ -157,25 +158,61 @@ func ReadStreamingConfig(file, dbName string) (StreamingCfg, error) {
 
 func getMysqlSourceStreamConfig(dbName string) *datastreampb.SourceConfig_MysqlSourceConfig {
 	mydb := &datastreampb.MysqlDatabase{
-		DatabaseName: dbName,
+		Database: dbName,
```

Comment:
Why is this renamed ?

Reply:
Because the updated library has these fields renamed